### PR TITLE
(Very)Simple Backup Script

### PR DESCRIPTION
**Pull request by Seji64**
I wrote a small and simple Backup Script (based on https://github.com/inverse-inc/sogo/blob/master/Scripts/sogo-backup.sh ). I know its far away from perfect, but i thought maybe some people may ask for it.

### CODE REVIEW

File: backup.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
+!/bin/bash
+
+DATE=$(date +"%Y%m%d_%H%M%S")
+BACKUP_DIR=/backups
```
Comment (mkuron) [inline backup.sh:4]: Make this a relative path inside the current directory and add it to .gitignore.

File: backup.sh [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+MAILCOW_DIR=/root/mailcow-dockerized
```
Comment (mkuron) [inline backup.sh:5]: Use `$(dirname $0)` as not everyone will be using this path.

File: backup.sh [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+function dumpit {
+  mkdir -m700  "$BACKUP_DIR/mailcow-${DATE}" 2>&1  | $LOG
+  if [ $? -ne 0 ]; then
+    exit 1
+  fi
+  cd $MAILCOW_DIR
+  source mailcow.conf
+  docker-compose exec -T mysql-mailcow mysqldump --default-character-set=utf8mb4 -u${DBUSER} -p${DBPASS} ${DBNAME} > backup_mysql.sql | $LOG
+  docker run --rm -i -v $(docker inspect --format '{{ range .Mounts }}{{ if eq .Destination "/var/vmail" }}{{ .Name }}{{ end }}{{ end }}' $(docker-compose ps -q dovecot-mailcow)):/vmail -v ${PWD}:/backup debian:stretch-slim tar cvfz /backup/backup_vmail.tar.gz /vmail | $LOG
```
Comment (mkuron) [inline backup.sh:50]: Most of the other volumes also need to be backed up, e.g. the Redis database, Postfix‘s queue, Rspamd‘s training state, ...

**Comment by cklabautermann** [comment]
Thanks for your work. could you please provied an easy way to restore a backup as well?
Thank you in advance!

File: backup.sh [hunk 4]
```diff
@@ -0,0 +1,67 @@
+!/bin/bash
```
Comment (max-baz) [inline backup.sh:1]: You forgot the `#` as the first character here.

**Comment by mkuron** [comment]
Closing this because author never followed up with requested changes. I will reopen it if you are still interested, @Seji64.

**Pull request closed without merging**